conf.d/halostatue_fish_docker.fish

```diff
@@ -8,7 +8,7 @@ function _halostatue_fish_docker_print_containers --description 'Print a list of
         case running
             set filter --filter status=running
```

Comment:
These should both be `set filter -- --filter status=running`. It’s my error for writing it wrong in the first place, but the first `--` is ignored and ensures that if Fish later adds a `--filter` flag to `set` we won’t have the wrong thing happen.

Reply:
Including the extra `--` causes the following error to be displayed:
```
"docker ps" accepts no arguments.
```

and it no longer suggests the containers. Looks like the `--` gets passed to the docker ps command if it's there.

Reply:
OK. Weird.